### ISSUE TITLE
GUI: don't get stuck when closing while a signing session is ongoing

### ISSUE THREAD

**Issue by darosior** [wizardsardine/liana#756]
Trying to close the GUI after clicking on "Sign" such as a connection to the signing device is established is going to hang until the connection to the signing device is closed.

**Comment by darosior** [comment]
Fairly related to #381, which @edouardparis recently investigated.

**Comment by pythcoiner** [comment]
Just hit the same issue while reviewing adress on signing device.

**Comment by edouardparis** [comment]
Not really, #381 is about sharing connection to ledger device with an other app wich sadly cannot be prevented unless showing an hint, while this issue is happening even in the case the gui is the only app talking to the ledger.

My main suspect is that the App cannot be dropped immediately by iced because in its internal, the Arc<HWI + Sync + Send> has still a reference counting in the Command. There is certainly a trick possible with tokio::task::AbortHandle, I am investigating.
